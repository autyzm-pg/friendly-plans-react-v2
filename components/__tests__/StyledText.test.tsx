import { shallow } from 'enzyme';
import React from 'react';

import { StyledText } from '../StyledText';

describe('StyledText', () => {
  const content = 'text content';

  it('should match snapshot', () => {
    const wrapper = shallow(<StyledText>{content}</StyledText>);
    expect(wrapper).toMatchSnapshot();
  });
});
