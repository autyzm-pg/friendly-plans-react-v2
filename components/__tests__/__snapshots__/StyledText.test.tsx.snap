// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledText should match snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Rubik-Regular",
      },
      undefined,
    ]
  }
>
  text content
</Text>
`;
